contracts/OptimizedForceMove.sol

```diff
@@ -381,6 +352,90 @@ contract OptimizedForceMove {
         channelStorageHashes[channelId] = keccak256(abi.encode(channelStorage));
     }
 
+    struct ChannelStorageLite {
+        uint256 finalizesAt;
+        bytes32 stateHash;
+        address challengerAddress;
+        bytes32 outcomeHash;
+    }
+
+    function respondWithAlternative(
+        FixedPart memory fixedPart,
+        uint256 largestTurnNum,
+        ForceMoveApp.VariablePart[] memory variableParts,
+        uint8 isFinalCount, // how many of the states are final
+        Signature[] memory sigs,
+        uint8[] memory whoSignedWhat,
+        bytes memory channelStorageLiteBytes
```

Comment:
Would be good to add a note about why you do this here (presumably "stack too deep").

Reply:
Addressed in ec82e8c